tools/registry.py

```diff
@@ -223,17 +223,21 @@ def get_all_modules(self):
     modules_dir = self.root.joinpath("modules")
     return [path.name for path in modules_dir.iterdir()]
 
-  def get_module_versions(self, module_name):
+  def get_module_versions(self, module_name, include_yanked=True):
     module_versions = []
     metadata = self.get_metadata(module_name)
     for version in metadata["versions"]:
       module_versions.append((module_name, version))
+    if not include_yanked and "yanked_versions" in metadata:
```

Comment:
Can you avoid adding the module version if the version is yanked instead of removing it afterwards?